captum/attr/_utils/summarizer.py

```diff
@@ -140,14 +140,14 @@ def _reorder_stats(stats):
     return stats, summary_stat_indexs
 
 
-def CommonSummarizer():
+def CommonSummarizer(summarizer_class=Summarizer):
```

Comment:
This feels a bit unusual. Could we probably Instead of having `CommonSummarizer` default all those arguments for the `Summarizer` ? The user can provide another list if they prefer to override it.
```
class Summarizer(stats=[Mean(), Var(order=1), StdDev(order=1), Min(), Max()]):

```
ClassSummarizer will then extend Summarizer since both are Summarizers and it makes sense that `ClassSummarizer` extends `Summarizer`. Let me know.

Reply:
I think I'll convert this function to return the stats used to construct the summarizer, i.e. `CommonStats()`; in-fact I'll move it to `stats.py`

I'll try to make ClassSummarizer extend Summarizer. I decided not to because it seemed like I could just use composition over inheritance.